src/web_app_skeleton/spec/setup/start_app_server.cr.ecr

```diff
@@ -5,6 +5,8 @@ spawn do
 end
 
 at_exit do
+  <%- if browser? -%>
   LuckyFlow.shutdown
+  <%- end -%>
```

Comment:
This  file was moved to always be added whether you have  API mode or "full", but we only want LuckyFlow is using "full" mode